src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Aapt2.targets

```diff
@@ -0,0 +1,168 @@
+		ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
+		AcwMapFile="$(_AcwMapFile)"
+		CustomViewMapFile="$(_CustomViewMapFile)"
+		AndroidConversionFlagFile="$(_AndroidStampDirectory)_ConvertLibraryResourcesCases.stamp"
+	/>
+	<Touch Files="$(_AndroidStampDirectory)_ConvertLibraryResourcesCases.stamp" AlwaysCreate="True" />
+</Target>
+
+<Target Name="_CompileAndroidLibraryResources" DependsOnTargets="_ConvertLibraryResourcesCases"
+		Condition="'$(_AndroidUseAapt2)' == 'True'"
+		Inputs="%(_LibraryResourceHashDirectories.StampFile)"
+		Outputs="$(_AndroidLibraryFlatArchivesDirectory)%(_LibraryResourceHashDirectories.Hash).stamp"
+	>
+	<MakeDir Directories="$(_AndroidLibraryFlatArchivesDirectory)" Condition="!Exists('$(_AndroidLibraryFlatArchivesDirectory)')" />
+	<Aapt2Compile
+		Condition=" '@(_LibraryResourceHashDirectories)' != '' "
+		ContinueOnError="$(DesignTimeBuild)"
+		ResourceDirectories="@(_LibraryResourceHashDirectories)"
+		ExplicitCrunch="$(AndroidExplicitCrunch)"
+		ExtraArgs="$(AndroidAapt2CompileExtraArgs)"
+		FlatArchivesDirectory="$(_AndroidLibraryFlatArchivesDirectory)"
+		ToolPath="$(Aapt2ToolPath)"
+		ToolExe="$(Aapt2ToolExe)"
+	/>
+	<Touch
+			Files="$(_AndroidLibraryFlatArchivesDirectory)%(_LibraryResourceHashDirectories.Hash).stamp"
+			AlwaysCreate="True"
+	/>
+	<ItemGroup>
+		<FileWrites Include="$(_AndroidLibraryFlatArchivesDirectory)%(_LibraryResourceHashDirectories.Hash).flata" />
+		<FileWrites Include="$(_AndroidLibraryFlatArchivesDirectory)%(_LibraryResourceHashDirectories.Hash).stamp" />
+	</ItemGroup>
+</Target>
+
+<Target Name="_CompileResources"
+		Condition="'$(_AndroidUseAapt2)' == 'True'"
+		Inputs="@(AndroidResource)"
+		Outputs="$(_AndroidLibraryFlatArchivesDirectory)\compiled.flata"
+	>
+	<MakeDir Directories="$(_AndroidLibraryFlatArchivesDirectory)" Condition="!Exists('$(_AndroidLibraryFlatArchivesDirectory)')" />
+	<!-- Change cases so we support mixed case resource names -->
+	<ConvertResourcesCases
+		ContinueOnError="$(DesignTimeBuild)"
+		ResourceDirectories="$(MonoAndroidResDirIntermediate)"
+		AdditionalResourceDirectories="@(_LibraryResourceDirectories)"
+		ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
+		AcwMapFile="$(_AcwMapFile)"
+		CustomViewMapFile="$(_CustomViewMapFile)"
+		AndroidConversionFlagFile="$(_AndroidLibraryFlatArchivesDirectory)\compiled.flata"
+	/>
+	<Aapt2Compile
+		ContinueOnError="$(DesignTimeBuild)"
+		ResourceDirectories="$(MonoAndroidResDirIntermediate)"
+		ResourceNameCaseMap="$(_AndroidResourceNameCaseMap)"
+		ExplicitCrunch="$(AndroidExplicitCrunch)"
+		ExtraArgs="$(AndroidAapt2CompileExtraArgs)"
+		FlatArchivesDirectory="$(_AndroidLibraryFlatArchivesDirectory)"
+		ToolPath="$(Aapt2ToolPath)"
+		ToolExe="$(Aapt2ToolExe)"
+	/>
+	<ItemGroup>
+		<FileWrites Include="$(_AndroidLibraryFlatArchivesDirectory)\compiled.flata" />
+	</ItemGroup>
+</Target>
+
+<Target Name="_GenerateJavaDesignerForComponentAapt2"
+  Condition="'$(_AndroidUseAapt2)' == 'True'"
```

Comment:
Attributes should be indented an "extra tab stop", i.e. 4 spaces from the left edge here, 6 spaces on line 146, etc.